rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -24,15 +49,24 @@ Bots can be licensed to use secure sessions. Some properties, like name, service
 
 |===
 
-== License Bots for Secure Sessions
+== Assign Sessions to Bots
 
-If bots should run processes in a hidden session, assign them secure session licenses.
+The bot runs a process configuration in a session on the computer where the bot is installed. 
+
+If bots should run processes in a hidden session, assign them at least one secure session.
 
 The number of secure sessions that can be assigned and served by the bot depends on the hardware of the computer on which the bot is installed
 Please refer to the system and hardware requirements topic to assess the maximum number of secure sessions for the bot.
 
+Enabling the same setting via the RPA Bot Configurator, limits but not not overwrites this setting.
+
 . In the *Bots* view of the *Bot Management* module, click *Secure Session Assignment*.
-. Set the number of sessions the bot may use.
+. Set the number of sessions the bot may use:
++
+* Session Assignment >= 1: 
+** The bot consumes up the the specified number of Secure Sessions.
+* Session Assignment = 0
+The bot consumes the desktop session.
```

Comment:
Is this correct? It sounds like it's eating the session! Should desktop be init cap? 
Should it be `** The bot...`